### ISSUE TITLE
pangolin::DisplayBase().RecordOnRender

### ISSUE THREAD

**Issue by lfwin** [stevenlovegrove/Pangolin#794]
this function 'RecordOnRender' is removed in this new version of Pangolin, which function can work as 'RecordOnRender' so can save screen displays to a video.

**Comment by stevenlovegrove** [comment]
Yeah, sorry about that - I had to remove it in order to decouple the libraries. There isn't a direct replacement at the moment.

The closet thing you can do is save the view to an image, which you can use to generate frames to convert to a video later.

pangolin::SaveWindowOnRender("window");
